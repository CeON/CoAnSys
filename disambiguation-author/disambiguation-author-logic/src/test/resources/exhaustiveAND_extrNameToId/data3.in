{(c1,Kowalski,[7#{(2011)},A#{(An Empirical Study on the AND)}]),(c2,Kowalski,[EX_DOES_NOT_EXIST#{(Unused information)},7#{(2011)},A#{(An Empirical Study on the AND)}]),(c3,Kowalski,[7#{(2011)},A#{()}]),(c4,Kowalski,[7#{(2011)},A#{}]),(c5,Kowalski,[7#{(2011)}])}	{(1,0,1.0),(2,0,0.0),(3,0,0.0),(4,0,0.0)}
