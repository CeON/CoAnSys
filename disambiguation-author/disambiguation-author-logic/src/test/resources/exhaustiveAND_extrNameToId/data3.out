(c803522c-d056-30be-a5ca-a4e552080ab5,{(c1),(c2),(c3),(c4),(c5)})
